### PR TITLE
[RND-286] - Avoid Logging payloads

### PR DESCRIPTION
**Pull request by stevenarnold-ed-fi**

### CODE REVIEW

File: Meadowlark-js/backends/meadowlark-opensearch-backend/src/repository/QueryOpensearch.ts [hunk 1]
```diff
@@ -101,8 +101,13 @@ export async function queryDocuments(request: QueryRequest, client: Client): Pro
 
     const { body } = await performSqlQuery(client, query);
     recordCount = body.total;
-    Logger.debug(`Result: ${JSON.stringify(body)}`, traceId);
+
     documents = body.datarows.map((datarow) => JSON.parse(datarow));
+
+    if (isDebugEnabled()) {
```
Comment (bradbanister) [inline Meadowlark-js/backends/meadowlark-opensearch-backend/src/repository/QueryOpensearch.ts:107]: Trying out wrapping "expensive" operations for logging in a log level check.

**Comment by stephenfuqua** [comment]
I found an interesting edge case in the log:

```
2022-09-12 12:05:38 info OwnershipSecurity.rejectByOwnershipSecurity /v3.3b/ed-fi/students {"action":"upsert","path":"/v3.3b/ed-fi/students","traceId":"87e206bd-57d3-46f5-beea-ec9acddf2cd8","headers":{"user-agent":"vscode-restclient","content-type":"application/json","authorization":"bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInJvbGVzIjpbInZlbmRvciJdLCJjbGllbnRfaWQiOiJtZWFkb3dsYXJrX2tleV8xIiwic3ViIjoic3VwZXItZ3JlYXQtU0lTIiwianRpIjoiMzVjNzY0MDgtMjMyNi00NGVlLThmZjktMjU2YjdhMzc2ZjUzIiwiaWF0IjoxNjYzMDAyMTMyLCJleHAiOjE2NjMwMDU3MzJ9.rOshGiMXQtQHftHrsWn4tS_p6O7axJWMhPuuzZr5404","accept-encoding":"gzip, deflate","content-length":"205","host":"localhost:3000","connection":"close","Host":"localhost:3000"},"queryParameters":{},"stage":"local","middleware":{"security":{"authorizationStrategy":"OWNERSHIP_BASED","clientId":"meadowlark_key_1"},"pathComponents":{"version":"v3.3b","namespace":"ed-fi","endpointName":"students"},"parsedBody":{"studentUniqueId":"s0zf6d1123d3e","firstName":"Hello","lastSurname":"World","birthDate":"2001-01-01","birthCountryDescriptor":"uri://ed-fi.org/CountryDescriptor#AD3"},"resourceInfo":{"projectName":"Ed-Fi","resourceVersion":"3.3.1-b","resourceName":"Student","isDescriptor":false},"documentInfo":{"documentReferences":[],"descriptorReferences":[{"projectName":"Ed-Fi","resourceName":"CountryDescriptor","documentIdentity":{"descriptor":"uri://ed-fi.org/CountryDescriptor#AD3"},"isDescriptor":true}],"documentIdentity":{"studentUniqueId":"s0zf6d1123d3e"},"superclassInfo":null},"headerMetadata":{"X-MetaEd-Project-Name":"Ed-Fi","X-MetaEd-Project-Version":"3.3.1-b","X-MetaEd-Project-Package-Name":"@edfi/ed-fi-model-3.3b"},"validateResources":true}}
```

**Comment by stephenfuqua** [comment]
Here's another, not-so-edge, case:

```
2022-09-12 12:07:82 info Upsert.upsert /v3.3b/ed-fi/students {"action":"upsert","path":"/v3.3b/ed-fi/students","traceId":"0ef4a79e-7a35-48cd-a324-33d9716ad898","headers":{"user-agent":"vscode-restclient","content-type":"application/json","authorization":"bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInJvbGVzIjpbImFzc2Vzc21lbnQiLCJ2ZW5kb3IiXSwiY2xpZW50X2lkIjoibWVhZG93bGFya19rZXlfNCIsInN1YiI6InRoZS1vbmUtc2lzIiwianRpIjoiOTMwZTM0NmQtMjcxMS00MjBjLWEwNDMtODAzMGRhOGNiN2QwIiwiaWF0IjoxNjYzMDAyMTQxLCJleHAiOjE2NjMwMDU3NDF9.Zrv_po3Hcrcipl0AJwGfKwFrF8PIJt2kimtmfOfz1ZA","accept-encoding":"gzip, deflate","content-length":"205","host":"localhost:3000","connection":"close","Host":"localhost:3000"},"queryParameters":{},"stage":"local","middleware":{"security":{"authorizationStrategy":"OWNERSHIP_BASED","clientId":"meadowlark_key_4"},"pathComponents":{"version":"v3.3b","namespace":"ed-fi","endpointName":"students"},"parsedBody":{"studentUniqueId":"s0zf6d1123d3e","firstName":"Hello","lastSurname":"World","birthDate":"2001-01-01","birthCountryDescriptor":"uri://ed-fi.org/CountryDescriptor#AD3"},"resourceInfo":{"projectName":"Ed-Fi","resourceVersion":"3.3.1-b","resourceName":"Student","isDescriptor":false},"documentInfo":{"documentReferences":[],"descriptorReferences":[{"projectName":"Ed-Fi","resourceName":"CountryDescriptor","documentIdentity":{"descriptor":"uri://ed-fi.org/CountryDescriptor#AD3"},"isDescriptor":true}],"documentIdentity":{"studentUniqueId":"s0zf6d1123d3e"},"superclassInfo":null},"headerMetadata":{"X-MetaEd-Project-Name":"Ed-Fi","X-MetaEd-Project-Version":"3.3.1-b","X-MetaEd-Project-Package-Name":"@edfi/ed-fi-model-3.3b"},"validateResources":false}}
```

**Comment by bradbanister** [comment]
I missed `parsedBody` in the request. 🙄

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**